nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -125,8 +129,7 @@ public void skipClass() {
 
   @Test
   public void skipNestedClass() {
-    compilationHelper
-        .setArgs(
+    makeTestHelperWithArgs(
```

Comment:
We need changes like this one because in EP 2.5.1 and beyond you can no longer call `setArgs` more than once on a `CompilationTestHelper` object

Reply:
Can we add a comment noting that restriction above `makeTestHelperWithArgs`?

Reply:
700f8527842a377ce2dfa0ef5fa421ae82e70f5f